admin_guide/sysctls.adoc

```diff
@@ -0,0 +1,168 @@
+[[admin-guide-sysctls]]
+= Sysctls
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Certain sysctl settings are exposed via Kubernetes, allowing users to modify
+kernel parameters at runtime for namespaces within a container. Only sysctls
+that are namespaced can be set independently on pods; if a sysctl is not
+namespaced (called _node-level_), it cannot be set within {product-title}.
+
+ifdef::openshift-enterprise[]
+[NOTE]
+====
+As of {product-title} 3.3.1, sysctl support is a feature in
+link:https://access.redhat.com/support/offerings/techpreview[Technology
+Preview].
+====
+endif::[]
+
+[[undersatnding-sysctls]]
+== Understanding Sysctls
+
+In Linux, the sysctl interface allows an administrator to modify kernel
+parameters at runtime. Parameters are available via the *_/proc/sys/_* virtual
+process file system. The parameters cover various subsystems such as:
+
+- kernel (common prefix: *_kernel._*)
+- networking (common prefix: *_net._*)
+- virtual memory (common prefix: *_vm._*)
+- MDADM (common prefix: *_dev._*)
+
+More subsystems are described in
+link:https://www.kernel.org/doc/Documentation/sysctl/README[Kernel documentation]. To get a list of all parameters, you can run:
+
+----
+$ sudo sysctl -a
+----
+
+[[namespaces-vs-node-level-sysctls]]
+== Namespaced vs Node-Level Sysctls
```

Comment:
Hmm ok there's "namespaced" in a heading now. Also, this doesn't match the anchor link. Should it?